backend/main_page/views/sentiment_analysis.py

```diff
@@ -0,0 +1,31 @@
+import requests
+from rest_framework.views import APIView
+
+from django.urls import reverse
+
+class SentimentAnalysis(APIView):
```

Comment:
Make the `SentimentAnalysis` class inherit the `View` class not `APIView`

~~`class SentimentAnalysis(APIView):`~~

`class SentimentAnalysis(View):`